changelog/4874.doc.rst

```diff
@@ -0,0 +1 @@
+Added a gallery example demonstrating zoomed image of map.
```

Comment:
I'm not sure what to change this to, but we should expand upon this example being just a zoomed image. It undersells it.

Reply:
Added a gallery example to show zoom out image of particular region on map.  

> It seems the doc build broke, any chance you can render the example locally and add a screenshot of it?
yeah totally, even i was thinking of it.